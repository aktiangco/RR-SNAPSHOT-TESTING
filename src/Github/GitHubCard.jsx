import Card from 'react-bootstrap/Card'
import silly from './images/silly-dog.jpeg'

const GitHubCard = () => {
    <div>
        <Card>
            <Card.Img src={silly} alt="silly" />
            <Card.Body>
                <Card.Title>Silly Me</Card.Title>
                <Card.Text>
                    Hello World, It's me just being silly.                
                </Card.Text>
            </Card.Body>
        </Card>
    </div>
}

export default GitHubCard