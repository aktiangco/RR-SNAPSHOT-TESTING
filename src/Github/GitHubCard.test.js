import GitHubCard from "./GitHubCard";
import renderer from "react-test-renderer";

// Challenge  #1
test('renders card component', () => {
    const tree = renderer.create(<GitHubCard />).toJSON()
    expect(tree).toMatchSnapshot()
  })