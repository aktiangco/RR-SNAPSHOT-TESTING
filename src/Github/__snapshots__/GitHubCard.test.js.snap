// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders card component 1`] = `null`;
